docs/guide/repository.md

```diff
@@ -26,39 +26,28 @@ The Public API exposes Symfony services for all of its Repository services.
 | `ezpublish.api.service.user`           | `eZ\Publish\API\Repository\UserService`          |
 | `ezpublish.api.service.user_preference`| `eZ\Publish\API\Repository\UserPreferenceService`|
 
-## SPI and API repositories
+## API
 
-The `ezpublish-api` and `ezpublish-spi` repositories are read-only splits of `ezsystems/ezpublish-kernel`
-They are available to make dependencies easier and more lightweight.
+Every Public API Service interface and Value object defined in `eZ\Publish\API` namespace strictly follows [Semantic Versioning](https://semver.org/) *backward compatibility (BC)* promise for API *consumers*.
+It means that every usage of API (API call) will work between minor releases.
```

Comment:
second-guessing myself: maybe instead of "will work" - "is guaranteed to work" - sounds more pro ;)